mjh_sanity/godas_obssanitycheck.py

```diff
@@ -0,0 +1,167 @@
+from datetime import datetime
+from netCDF4 import Dataset, num2date, date2num
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+from datetime import datetime, timedelta
+from os.path import exists
+import matplotlib.pyplot as plt
+import numpy as np
+class Sanity:
```

Comment:
You're doing a sanity check, might as well call your class that as well. `SanityCheck` or `ObsSanityCheck` or `ObsCheck` ... Just something more meaningful than `Sanity`.